src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let number = 0;
+
+  if (days < 3) {
+    number = days * 40;
+  } else if (days >= 3 && days < 7) {
+    number = days * 40 - 20;
+  } else if (days >= 7) {
+    number = days * 40 - 50;
+  }
+
+  return number;
```

Comment:
The function name `calculateRentalCost` is descriptive and good. However, the variable `number` is not descriptive of its purpose. Consider renaming it to something more descriptive, like `totalCost`.